.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,11 +15,10 @@ Please include a short resume of the changes and what is the purpose of PR. Any
 
 ## Target serie
 
-- [ ] 2.8.x
```

Comment:
the 2.8.x needs to be kept for security issues. Please check with the CTO before